schema.sql

```diff
@@ -38,4 +38,31 @@ ALTER TABLE animals ADD FOREIGN KEY (species_ID) REFERENCES species(id);
 /*Add column owner_id which is a foreign key referencing the owners table*/  
 
 ALTER TABLE animals ADD COLUMN owners_ID INT;
-ALTER TABLE animals ADD FOREIGN KEY (owners_ID) REFERENCES owners(id);
\ No newline at end of file
+ALTER TABLE animals ADD FOREIGN KEY (owners_ID) REFERENCES owners(id);
+
+
+/*Create vets table*/
+
+CREATE TABLE vets (
+    id INT GENERATED ALWAYS AS IDENTITY, 
+    name CHAR(75), age INT, 
+    date_of_graduation DATE, 
+    PRIMARY KEY(id)
+);
+
+/*Create table specializations */
+
+CREATE TABLE specializations (
+    vets_id INT REFERENCES vets(id) , 
+    species_id INT REFERENCES species(id), 
+    PRIMARY KEY(species_id, vets_id)
```

Comment:
- [ ] There should be only one primary key to represent unique, Kindly removes it, There is no requirement for uniqueness here.